src/atoms/link.tsx

```diff
@@ -0,0 +1,22 @@
+import Link from '@mui/material/Link';
+
+interface UnderlineLinkProps {
+    children: string;
+    linkType?: 'primary' | 'secondary' | 'default';
+    underline?: 'none' | 'hover' | 'always';
+}
+
+function UnderlineLink({ children, linkType = 'default', underline = 'none' }: UnderlineLinkProps) {
```

Comment:
This component should be named Link not UnderlineLink

Reply:
corrected